src/cli.rs

```diff
@@ -252,6 +252,20 @@ fn build_command() -> Command<'static> {
                      when trying to benchmark output speed.",
                 ),
         )
+        .arg(
+            Arg::new("output")
+                .long("output")
+                .conflicts_with("show-output")
+                .takes_value(true)
+                .value_name("WHERE")
+                .help(
+                    "Control where the output of the benchmark is redirected.\n\n  \
+                     null: Redirect output to /dev/null (the default).\n\n  \
+                     pipe: Feed the output through a pipe before discarding it.\n        \
+                     Useful if the benchmark special-cases /dev/null.\n\n  \
```

Comment:
I'm not sure if most users would understand the "Useful if the benchmark special-cases /dev/null" part. It would be great if we could try to make this easier somehow. Maybe by writing under `null`: "Note that some programs like 'grep' detect that the output goes to /dev/null, allowing them to enable certain optimizations. If you want to avoid that, consider using '--output=pipe'".

Reply:
That's a good idea, I'll update the help text like that.